### ISSUE TITLE
Change `<h1>Archival Collections at Institution</h1>` to a `<span>`

### ISSUE THREAD

**Issue by jvine** [projectblacklight/arclight#821]
Part of #820 

On all but the home page, this heading does not communicate the content of the page: it's branding content. It's also contained in the `<header>` section, while the page `<h1>` should ideally be in the main content section.

Tasks:
- [x] No change to home page!
- [x] For all other pages, change `<h1><%= t('arclight.masthead_heading') %></h1>` to `<span><%= t('arclight.masthead_heading') %></span>`
- [x] Apply the css for `.al-masthead {  h1 ` to `.al-masthead {  span`  
- [x] Add `display: block` to the span

"After" should be visually identical to "before".
